### ISSUE TITLE
Issue when the a document with ID is deleted in ElasticSearch and ESConnector is publishing

### ISSUE THREAD

**Issue by rrangine** [confluentinc/kafka-connect-elasticsearch#145]
Hi,

I am not sure if this issue is logged earlier but. Here is the issue that I am facing with:

Using Kafka 0.11, Connect ES connector, ElasticSearch 5.2

1. A json document is publish into a kafka topic with Key:"1" and Value: Json doc
2. ESConnector is listeing on this Kafka Topic and has created an based on the connector config.
3.Can see the indexed document in ES
4. Deleted the document in ES index by ID: 1, manuall( curl)
5. Published a new version of json document into Kafka Topic with same ID:1 
6. Do not see that document is being published into the ES Index.
7. Checked the Kafka Consumer lag on this ESConnector consumer group-  lags :0 on all 3 workers. Logsize and Offsets are same for all the pid's(3 workers)
8. Published **another** version of JSon  document into Kafka Topic with same ID:1
9. Now can see the document indexed in ES.
10. Checked the Kafka Consumer lag on this ESConnector consumer group-  lags :0 on all 3 workers. Logsize and Offsets are same for all the pid's(3 workers).

So, in short, I am missing the version of document at step #5. Any ideas what I am missing here OR someone can duplicate this and let me know if indeed this is a bug.

Thanks,
Raja Rangineni

**Comment by vecernik** [comment]
any plans for this one? Adding transforms.unwrap.drop.tombstones=false doesn't help.
